tensorflow-tools/src/test/java/org/tensorflow/tools/ndarray/impl/sequence/ElementSequenceTest.java

```diff
@@ -37,8 +41,7 @@ public void iterateVectorsWithIndex() {
     IntNdArray array = NdArrays.ofInts(Shape.of(2, 3, 2));
 
     @SuppressWarnings("unchecked")
-    NdArraySequence<IntNdArray> sequence = ElementSequence
-        .create((AbstractNdArray<Integer, IntNdArray>)array, 1);
+    NdArraySequence<IntNdArray> sequence = new SlicingElementSequence((AbstractNdArray<Integer, IntNdArray>)array, 1);
```

Comment:
Do we still need to suppress warnings here? It looks like it's all properly typed?

Reply:
good catch!